src/components/Chart.jsx

```diff
@@ -68,8 +72,8 @@ class Chart extends Component {
 
         const contextClassName = () => {
             let className = '';
-            className += isMobile ? 'smartcharts-mobile' : '';
-            className += (typeof theme === 'string' ) ? ` smartcharts-${theme}` 
+            className += (isMobile === undefined || isMobile) ? 'smartcharts-mobile' : '';
```

Comment:
nitpick: if you need a default value for a prop use es6 destructuring default values.

`const { ... isMobile = true, ...} = this.props;`